### PR TITLE
Update UserTypeForm.tsx

### LINKED ISSUE #134: Clear user type when re-opening sign-up page
Expected:
When (re)opening the sign-up page the user type field should be empty so that the user can select any item from the list

Actual:
When opening the sign-up page, the user type field is pre-filled 
The user can only select any other option, not the currently filled option 

https://user-images.githubusercontent.com/2820084/156596506-f4d6f726-7c6c-461b-9b37-cf72148f9812.mov

### PR DESCRIPTION
**Pull request by idormenco**
close #134

### CODE REVIEW

File: components/UserTypeForm/UserTypeForm.tsx [hunk 1]
```diff
@@ -13,9 +13,10 @@ const UserTypeForm = () => {
   const [userType, setUserType] = useState<number | null>()
   const dispatch = useDispatch()
   const { data } = useUserTypeForm()
-  const userTypeOptions = data?.type.choices || []
+  const emptyOption = { value: 0, display_name: '' }
+  const userTypeOptions = [emptyOption, ...(data?.type.choices || [])]
```
Comment (cipick) [inline components/UserTypeForm/UserTypeForm.tsx:17]: can you explain why is this needed? i fixed the issue with the dropdown if that was your intention. the problem was that `Dropdown` used the label as defaultValue..

Comment (idormenco) [reply components/UserTypeForm/UserTypeForm.tsx:17]: this was my fix for the issue with dropdown. I will close this pr
